unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Definitions/Animator.proto

```diff
@@ -0,0 +1,18 @@
+﻿syntax = "proto3";
+
+import "common/id.proto";
+option (ecs_component_id) = 1042;
+
+message PBAnimator {
+   repeated PBAnimationState states = 1;
+}
+
+message PBAnimationState {
+  string name = 1;
+  string clip = 2;
+  bool playing = 3;
+  float weight = 4;
+  float speed = 5;
+  bool loop = 6;
+  bool should_reset = 7;
```

Comment:
I think we should analyze if `should_reset` field applies to the data oriented components or if we should just rename it `reset`?
because, if I recall correctly, it was a hidden field set by the `Play()` method in previous ecs versions

Reply:
We can change that kind of name when we want. I think that we should take a look at all components names